src/main/java/eu/dissco/backend/repository/ElasticSearchRepository.java

```diff
@@ -140,18 +155,18 @@ private Instant parseDate(JsonNode instantNode) {
     return null;
   }
 
-  private String getText(JsonNode digitalSpecimen, String element) {
-    var jsonNode = digitalSpecimen.get(element);
+  private String getText(JsonNode annotation, String element) {
```

Comment:
Do we use this function genericly? Maybe just call it jsonNode or node.

Reply:
removed altogether

Reply:
Even better